### ISSUE TITLE
All wda  mac interface sets

### ISSUE THREAD

**Issue by HaoWShi** [appium/appium-mac2-driver#283]
I would like to know where I can see all the current set of interfaces? anybody know?

**Comment by KazuCocoa** [comment]
do you mean https://github.com/appium/appium-mac2-driver/blob/master/WebDriverAgentMac/WebDriverAgentLib/Commands/AMActionCommands.m for example?
Then https://github.com/appium/appium-mac2-driver/tree/master/WebDriverAgentMac/WebDriverAgentLib/Commands

**Comment by HaoWShi** [comment]
I have read the files in https://github.com/appium/appium-mac2-driver/tree/master/WebDriverAgentMac/WebDriverAgentLib/Commands. 
In fact I mean https://github.com/appium/appium-mac2-driver/blob/master/lib/commands/app-management.js  , there are interface I can use. 
The interface you mentioned, how should I use it?

**Comment by KazuCocoa** [comment]
The WDA's APIs are expected to be used via mac2 driver running on a host machine, so https://github.com/appium/appium-mac2-driver/blob/master/lib/commands/app-management.js like you addresses are endpoints.

Each method names defined in JavaScript code are mapped in https://github.com/appium/appium/blob/master/packages/base-driver/lib/protocol/routes.js . The Web API is defined in https://appium.io/docs/en/latest/commands/base-driver/ or https://appium.io/docs/en/latest/guides/execute-methods/ https://github.com/appium/appium-mac2-driver?tab=readme-ov-file#platform-specific-extensions

**Comment by HaoWShi** [comment]
Yes, It is what I want, I will read them firstly. Thank you very much
By the way, I noticed the locked/unlock/lock interface on the phone, but on the Mac, I don't see it. Is it not supported? Or is it another interface?

**Comment by KazuCocoa** [comment]
XCUITest driver achieves it with https://developer.apple.com/documentation/xctest/xcuidevice?language=objc but it looks like the documentation does not say macOS support it. So potentially macOS has no method to lock the screen over XCTest framework

**Comment by KazuCocoa** [comment]
Instead, for lock, I wondered if `Control-Command-Q: Immediately lock your screen.` works
https://support.apple.com/en-us/HT201236
https://github.com/appium/appium-mac2-driver?tab=readme-ov-file#macos-keys
